integration-tests/integration/prisonerProfile/prisonerLocationHistory.spec.js

```diff
@@ -268,91 +268,4 @@ context('Prisoner location history', () => {
         .contains('A long comment about what happened on the day to cause the move.')
     })
   })
-
-  context('Cell move comment functionality', () => {
```

Comment:
Why have you removed this integration test?

Reply:
I added this test before based on the wrong assumption that we could get an empty comment back, we only default on 404 and the system doesn't allow empty comment just like it doesn't allow over 4000 chars

Reply:
So should this integration test be tweaked to trigger the 404? That way the test is still valid.

Reply:
these tests already exist please see above lines 209 and 219

Reply:
Cant see anything in the integration file that relates to defaults. Looked at the following lines 209 and 219.

Reply:
the test that was removed asserts that if a rest call is made and the returned comment is of zero length then we default since this would not happen the test was removed. However, if this could happen? I'm happy to reinstate the test and make further code changes to support that behavior. The only time we default is when we get a 404 which are covered test on lines 209 and 219